tests/prague/eip2537_bls_12_381_precompiles/conftest.py

```diff
@@ -126,12 +126,18 @@ def call_contract_code(
     assert call_opcode in [Op.CALL, Op.CALLCODE, Op.DELEGATECALL, Op.STATICCALL]
     value = [0] if call_opcode in [Op.CALL, Op.CALLCODE] else []
 
+    precompile_gas_value_opcode: int | Op
+    if precompile_gas is None:
+        precompile_gas_value_opcode = Op.GAS
+    else:
+        precompile_gas_value_opcode = precompile_gas + precompile_gas_modifier
```

Comment:
Was this required for the fix we needed? Or an coherency improvement?

Reply:
The problem was that the pytest fixtures try to automatically calculate the precompile_gas needed for every precompile gas.

But in this case it was done in reverse so we needed the call contract without knowing the gas, so this is a fix to allow creation of the contract without knowing the gas by simply passing the full available gas to the precompile in the call.